### DISCUSSION TITLE
Which release of Project Reunion will support XBOX UWP?

### DISCUSSION THREAD

**Discussion by sharpninja** [microsoft/WindowsAppSDK#668] [General] [answered]
I've had to put all my XBOX plans on the back burner waiting for WinUI 3 and WebView 2 on XBOX.  When is that coming?

**Comment by ghost** [comment]
~ _maybe_ ~ have to wait until Windows App Container comes to Win32 apps. 🙂

**Comment by tfennel** [comment, marked as answer]
Hey @sharpninja, we'd love to learn more about your XBOX plans specifically and what you're building there and why WinUI/WV2 helps you. Understanding your jobs to be done helps us learn! 

This year our core focus is on Desktop, but that doesn't mean we don't want to hear about requirements for other platforms!

**Reply by sharpninja** [reply]
Anything having to do with streaming video is a natural fit for Xbox.  I actually stopped working on a streaming video search client because the UI is not tolerant of odd images that don't show up in a certain timeframe and caching them ahead of time causes long pauses waiting to cache the images or just let the Image load straight from the URL which causes oddities with grid layout until, or worse, it will show an old version of the image if the image times out.  I want to use Blazor to create the UI instead and run it on the Xbox.

**Reply by nickrandolph** [reply]
Is there a reason you want a native app at all for Xbox? Why not point to at an xbox optimised website directly from the manifest file. Your website can access the WinRT apis allowing you to do all the things you need to for video (caching, drm etc).

**Reply by sharpninja** [reply]
Websites cost money to host, which I don't need to spend if I have a native app.  I am paying for the data that feeds the app and for a DB server for the app, but I shouldn't have to buy more cloud services just because Xbox is in Limbo right now.
